Components/Avatar/avatar.css

```diff
@@ -0,0 +1,33 @@
+* {
```

Comment:
As this is common for all the files. Put this in a separate CSS and then import to use.

Reply:
Ok, I thought it was just 2 lines so an import was not needed. Thanks